tap_pipedrive/tap.py

```diff
@@ -131,7 +131,7 @@ def do_sync(self, catalog):
 
                     stream.update_endpoint(deal_id)
                     stream.start = 0   # set back to zero for each new deal_id
-                    self.do_paginate(stream)
+                    self.do_paginate(stream, stream_metadata)
```

Comment:
Where is stream_metadata defined?  I think you may need to repeat lines 147 & 148 above this?

Reply:
@nick-mccoy It was being assigned by the previous loop passes, which didn't come up in tests. I just moved the assignment outside of the if/else since both conditions use it.